src/models/Product.ts

```diff
@@ -0,0 +1,101 @@
+import {
+  AllowNull,
+  BelongsTo,
+  Column,
+  DataType,
+  ForeignKey,
+  Model,
+  PrimaryKey,
+  Table,
+} from 'sequelize-typescript';
+import { Phone } from './Phone';
+
+@Table({
+  tableName: 'products',
+  createdAt: false,
+  updatedAt: false,
+})
+export class Product extends Model {
```

Comment:
Looks like the Phone model is the same as Product
Maybe we can leave only the Product model with all fields and provide a category field with an enum type (Phone/Tablet/Accessories)

Reply:
Let's combine two models into one